update: circuit framework added (#194)
JetBrains/kotlin-multiplatform-dev-docs: topics/compose/compose-navigation-routing.md

|-----------------------------------------------------|-----------------------------------------------------------------------------------------------------------------------------------------------------------------|
| [Voyager](https://voyager.adriel.cafe)              | A pragmatic approach to navigation                                                                                                                              |
| [Decompose](https://arkivanov.github.io/Decompose/) | An advanced approach to navigation that covers the full lifecycle and any potential dependency injection                                                        |
| [Appyx](https://bumble-tech.github.io/appyx/)       | Model-driven navigation with gesture control                                                                                                                    |
| [PreCompose](https://tlaster.github.io/PreCompose/) | A navigation and view model inspired by Jetpack Lifecycle, ViewModel, LiveData, and Navigation                                                                  |

> | [Circuit](hhttps://slackhq.github.io/circuit/)      | A lightweight and extensible Compose framework for building Kotlin applications. It uses a custom Compose-based backstack implementation for navigable contents |

Reviewer: ```suggestion
| [Circuit](https://slackhq.github.io/circuit/)      | A lightweight and extensible Compose framework for building Kotlin applications, with a Compose-based backstack for navigation |
```
